### PR TITLE
[CAPI] Reflect privilege reviews

### PR DESCRIPTION
**Pull request by zhoonit**
- [CAPI] Reflect privilege reviews

```
This patch update document to add privilege to `ml_train_model_save()`
and `ml_train_model_load()`

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

File: api/capi/include/nntrainer.h [hunk 1]
```diff
@@ -576,14 +583,19 @@ int ml_train_model_save(ml_train_model_h model, const char *file_path,
  * configurations. Unless stated otherwise, ml_train_model_compile() has to
  * be called upon the @a model before calling this function.
  * @since_tizen 6.5
- *
+ * @remarks If you want to access only internal storage by using this function,
+ * you should add privilege %http://tizen.org/privilege/mediastorage. Or, if you
+ * want to access only external storage by using this function, you should add
+ * privilege %http://tizen.org/privilege/externalstorage. If you can access both
```
Comment (kparichay) [inline api/capi/include/nntrainer.h:589]: if you can -> if you want to ?

File: api/capi/include/nntrainer.h [hunk 2]
```diff
@@ -576,14 +583,19 @@ int ml_train_model_save(ml_train_model_h model, const char *file_path,
...
+ * storage, you must add all privilege.
```
Comment (kparichay) [inline api/capi/include/nntrainer.h:590]: all privilege -> all the privileges ?

**Review by kparichay — APPROVED** [review]
LGTM other than some minor comments

File: api/capi/include/nntrainer.h (hunk 1 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: updated :)

File: api/capi/include/nntrainer.h (hunk 2 above)
Comment (zhoonit) [reply api/capi/include/nntrainer.h]: done!

**Comment by zhoonit** [comment]
This PR will be until the privilige review is finished from the reviewer.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**